include/boost/geometry/algorithms/detail/envelope/initialize.hpp

```diff
@@ -0,0 +1,85 @@
+// Boost.Geometry (aka GGL, Generic Geometry Library)
+
+// Copyright (c) 2015, Oracle and/or its affiliates.
+
+// Contributed and/or modified by Menelaos Karavelas, on behalf of Oracle
+
+// Licensed under the Boost Software License version 1.0.
+// http://www.boost.org/users/license.html
+
```

Comment:
The license (last 2 lines) in this PR has two versions (sometimes 3 lines).
But apparently somehow we changed the license from a 3-line version to this 2-line version. Because the 3-lines version occurs 825 times, in our code the 2-lined version < 100 times.

This version is incorrect because it refers to the license guidelines, and not to the license itself.

I just checked the Boost page and they (now) recommend yet another version
http://www.boost.org/users/license.html

So we should adapt our code to have only one version, and the commended one (I'm only talking about the last two lines, not about the rest)

So it now should read:
```// Distributed under the Boost Software License, Version 1.0.```
```//    (See accompanying file LICENSE_1_0.txt or copy at```
```//          http://www.boost.org/LICENSE_1_0.txt)```